_locales/en/messages.json

```diff
@@ -1063,13 +1063,17 @@
     "description": "Shown in a dialog to inform user that we experienced an unrecoverable error"
   },
   "icu:attachmentSaved": {
-    "messageformat": "Attachment saved.",
+    "messageformat": "{count, plural, one {Attachment} other {# attachments}} saved",
```

Comment:
Sorry, I should have told you this up front. We need this to be a new string - maybe `icu:attachmentSavedPlural`? The old string needs to be left there for full release flexibility.

Reply:
No problem, I will change this.
But I should use the new `icu:attachmentSavedPlural` in the ToastManager, correct?

Reply:
Yes, use the new string, but keep the old string around so hotfixes and such have access to it.